src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/DiagnosticComputer.cs

```diff
@@ -238,6 +238,14 @@ async Task<CompilationWithAnalyzersCacheEntry> GetOrCreateCacheEntryAsync()
             }
         }
 
+        /// <summary>
+        /// List of assemblies that contain analyzers that only run in-proc and should not be loaded OOP.
+        /// </summary>
+        private static readonly ImmutableArray<string> s_wellKnownAssembliesContainingIdeAnalyzersOnly = ImmutableArray.Create(
```

Comment:
Hmm, do we actually load those assemblies in OOP? (i thought so, but the integration tests are passing now so I'm confused)